components/general/CommonQuestions.tsx

```diff
@@ -0,0 +1,43 @@
+import styles from "@/styles/ContactPage.module.css";
+import { fontSyne, fontRobotoMono, fontSatoshi } from "@/pages/_app";
+import QuestionItem from "./QuestionItem";
+
+const CommonQuestions = () => {
+  return (
+    <section className={styles.questions}>
+      <div className={styles.questions_title__contrainer}>
+        <p
+          className={`${styles.msg_left_title__heading} ${fontRobotoMono.className} global__text-meta`}
+        >
+          FAQ
+        </p>
+        <h1 className={`global__heading--small ${fontSyne.className}`}>
+          Common
+          <br />
+          Questions
+        </h1>
+      </div>
+      <div className={styles.questions_right__container}>
+        <div className={styles.questions_right__question}>
+          <QuestionItem question="Integer vitae justo eget magna?" />
+          <p className={`${fontSatoshi.className} global__text-default ${styles.question__text}`}>
```

Comment:
![image](https://user-images.githubusercontent.com/73293255/222445396-12b0c2f6-79f6-4f9f-be79-f3ecf0c36a5e.png)
Try to include this in the Question Item component

Reply:
Good suggestion, I have done this.